templates/base.html

```diff
@@ -60,12 +69,8 @@
 <div class="container-fluid">
     <div class="row-fluid">
         <div class="col-md-12">
-            {% for message in messages %}
-                <div class="alert alert-{{ message.tags }}">{{ message }}</div>
-            {% endfor %}
-
-            {% block content %}
-            {% endblock %}
+          {% block content %}
```

Comment:
Die flash messages sind absichtlich in `base.html`, damit man diese nicht immer in den jew. Views angeben muss. Ist `messages` leer, wird eben nichts angezeigt. Oder hat es einen anderen Grund?